SPECS/clang/clang.spec

```diff
@@ -61,41 +68,55 @@ make DESTDIR=%{buildroot} install
 cd build
 make clang-check
 
-%clean
+%{clean}
 rm -rf %{buildroot}/*
 
 %files
 %defattr(-,root,root)
 %license LICENSE.TXT
 %{_bindir}/*
 %{_libexecdir}/*
-%{_libdir}/*.so.*
 %{_datadir}/*
 
+%files libs
+%defattr(-,root,root)
```

Comment:
There are likely packages in Base and Extended that either have a build requires or a requires with Clang.  Have you verified that they are unaffected by this change?